### DISCUSSION TITLE
2022/09/12/cf1120d/

### DISCUSSION THREAD

**Discussion by giscus** [DeCalvin2006/blog#4] [General]
# 2022/09/12/cf1120d/

题面链接

https://decalvin.netlify.app/2022/09/12/cf1120d/

**Comment by shiziping** [comment]
# $%%%$  $DeCalvin$ 太强了

**Reply by DeCalvin2006** [reply]
$\text{stO} \text\huge{SZP AK CSP-J} \text{Orz}$
